### DISCUSSION TITLE
Exporting Location coordinates

### DISCUSSION THREAD

**Discussion by david95p** [sepinf-inc/IPED#1430] [Q&A] [answered]
Hello, 
I'm trying to export location coordinates from an UFDR image. The exporting of the category 'Location' seems to work properly but in the exported files I don't get the coordinates (latitude, longitude). 
I'm currently running the IPED 4.0.6 on Ubuntu 20.04, beforheand I made some attempts with IPED 3.18.15 and worked as expected. 

The changes I've made are in IPED 4.0.6:
- enableAutomaticExportFiles = true in IPEDConfig.txt
- uncomment 'Locations' in conf/CategoriesToExport.txt

Result:

> ufed:Source: Apple Maps
ufed:TimeStamp: _time_
ufed:decoding_confidence: High
ufed:extractionId: 0
ufed:extractionName: Advanced Logical
ufed: id: _blank_45
ufed:isrelated: False
ufed:source_index: 10719

The changes I've made are in IPED 3.18.15:
- uncomment 'Locations' in profiles/[local]/default/conf/CategoriesToExport.txt

Result:

> ﻿ufed:Latitude: X
ufed:Longitude: Y
ufed:Source: Apple Maps
ufed:TimeStamp: _time_
ufed:decoding_confidence: High
ufed:extractionId: 0
ufed:extractionName: Advanced Logical
ufed: id: _blank_45
ufed:isrelated: False
ufed:source_index: 10719

What am I missing? 
Is there a better approach to extract locations to be processed (same applies for contacts in my case)?

Many thanks

**Comment by lfcnassif** [comment, marked as answer]
Sorry for the delay and thanks for reporting. Just opened and fixed #1433. Locations now are exported as:
`common:geo:locations: X;Y`
